meshers/transvoxel/transvoxel.cpp

```diff
@@ -1237,7 +1237,7 @@ void build_transition_mesh(
 									cell_textures.weights[index_vertex_a];
 							const FixedArray<uint8_t, MAX_TEXTURE_BLENDS> weights1 =
 									cell_textures.weights[index_vertex_b];
-							FixedArray<uint8_t, MAX_TEXTURE_BLENDS> weights;
+							FixedArray<uint8_t, MAX_TEXTURE_BLENDS> weights = {};
```

Comment:
What is this doing? Initializing earlier is pointless because the following for loop is doing it.

Reply:
Seems like MSVC being dumb
```
D:\Godot\Github\godot\modules\voxel\meshers\transvoxel\transvoxel.cpp(1246) : error C2220: the following warning is treated as an error
D:\Godot\Github\godot\modules\voxel\meshers\transvoxel\transvoxel.cpp(1246) : warning C4701: potentially uninitialized local variable 'weights' used
```
Looks like there's already some contingency in place for `fixed_array.h` (lines 56-61), so this might be in a similar boat?

Reply:
For this and the other 3 that didn't have any real "clean" resolution, I added an MSVC define to the top for handling these cases. Couldn't surround the affected areas with push/pop wrappers for some reason, but they're not header files to begin with so it's an acceptable fallback imo